src/components/Shell/ShellAppBar.tsx

```diff
@@ -95,30 +100,40 @@ export const ShellAppBar = ({
         >
           {title}
         </Typography>
-        <Tooltip title="Copy current URL">
-          <IconButton
-            size="large"
-            edge="start"
-            color="inherit"
-            aria-label="Copy current URL"
-            sx={{ ml: 'auto' }}
-            onClick={onLinkButtonClick}
-          >
-            <LinkIcon />
-          </IconButton>
-        </Tooltip>
         {doShowPeers ? (
-          <Tooltip title="Click to show peer list">
-            <IconButton
-              size="large"
-              edge="end"
-              color="inherit"
-              aria-label="Peer list"
-              onClick={onPeerListClick}
-            >
-              <StepIcon icon={numberOfPeers} />
-            </IconButton>
-          </Tooltip>
+          <Box display="inline-flex">
```

Comment:
Is this additional `Box` necessary? I was able to replace it with a React Fragment `<></>` and didn't see any change at runtime. Can it be removed?

In case we do need this: for consistency and extensibility, I'd like to stick with `sx` props for styling of MUI components.

```suggestion
          <Box sx={{ display: 'inline-flex' }}>
```

Reply:
Good point, this extra `Box` wasn't necessary and was replaced with a fragmnet